### PR TITLE
liwang profile branch

### PR DESCRIPTION
**Pull request by elaineliwang**
Submitting profile information for Li Wang

### CODE REVIEW

**Comment by ianabc** [comment]
Hi Elaine,

In my notes for the first part of the class I should have been clearer in how to make the replacements in _index.md. Your variables shouldn't be surrounded by angle brackets, e.g. do
```
role: Graduate Student
```
NOT
```
role: <GRADUATE STUDENT>
```
i.e. remove the < and the >. You can just update the file in that branch and it should automatically update the PR. This is a pretty standard thing, commonly you'll want to make minor changes after initially opening the PR based on feedback from the maintainers or improvements you've spotted. You can keep making changes up until the PR is merged. After it is merged you can always open another PR with any extra changes

Also, can you delete the `.DS_Store` file in this PR, it isn't needed.

File: content/authors/elaineliwang/_index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+social:
+- icon: twitter
+  icon_pack: fab
+  link: 
+- icon: github
+  icon_pack: fab
+  link: 
+user_groups:
+- committee: members
+  role: Project Member
```
Comment (ianabc) [inline content/authors/elaineliwang/_index.md:41]: Can you change "Project Member" to say what project you are in, e.g. change it to "ATCO Project Member" if you are in the ATCO project.

File: content/authors/elaineliwang/_index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# A list of your interests, add as many as you want
+interests:
+-  Partial Differential Equations
+-  
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: B.Sc in Mathematics
```
Comment (ianabc) [inline content/authors/elaineliwang/_index.md:28]: I just made up a degree here (a B.Sc from UBC), could you edit it to something relevant to you?

File: content/authors/elaineliwang/_index.md (hunk 1 above)
Comment (elaineliwang) [reply content/authors/elaineliwang/_index.md]: Thanks! @ianabc it is fixed!

File: content/authors/elaineliwang/_index.md (hunk 2 above)
Comment (elaineliwang) [reply content/authors/elaineliwang/_index.md]: Thanks! @ianabc it is fixed!

**Comment by ianabc** [comment]
That looks great! Thank you. I'll merge it in now.

**Pull request merged**